test/functional/feature_framework_startup_failures.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python3
+# Copyright (c) 2025-present The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""
+Verify framework startup failures only raise one exception since
+multiple exceptions being raised muddies the waters of what actually
+went wrong. We should maintain this bar of only raising one exception as
+long as additional maintenance and complexity is low.
+
+Test relaunches itself into child processes in order to trigger failures
+without the parent process' BitcoinTestFramework also failing.
+"""
+
+from test_framework.util import rpc_port
+from test_framework.test_framework import BitcoinTestFramework
+
+from hashlib import md5
+import re
+import subprocess
+import sys
+import time
+
+class FeatureFrameworkStartupFailures(BitcoinTestFramework):
+    def set_test_params(self):
+        self.num_nodes = 1
+
+    def setup_network(self):
+        # Don't start the node yet, as we want to measure during run_test.
+        self.add_nodes(self.num_nodes, self.extra_args)
+
+    # Launches a child test process running this same file, but instantiating
+    # a child test, and verifies that we (only) raise the expected exception.
```

Comment:
Maybe we could make the tenses more uniform here:
```suggestion
    # Launches a child test process that runs this same file, instantiates
    # a child test, and verifies that it raises only the expected exception.
```

Reply:
Adjusted in similar direction.